### DISCUSSION TITLE
Make nginx a core and installed feature when deploying swizzin (no more IP:PORT)

### DISCUSSION THREAD

**Discussion by userdocs** [swizzin/swizzin#603] [Feature Requests]
So this may as well start as a discussion to hammer out the potential benefits and problems that may arise from this idea.

I suggest that nginx is made part of the default installation of swizzin. Primarily this lead to these things.

- All applications have a simplified installation with a uniform approach to apps and we do not need to account for a non nginx environment (using ports, ssl and more tricky stuff to manage across apps)
- the panel could be to be moved to `/panel` by default
- Authelia can handle authentication across all applications and provide a more configurable end user option for authentication.

Ultimately I feel it would lead to a much more uniform and simplified process.

- Easier to make and add new apps
- Easier to link the main user account into unified authentication.

What are peoples thoughts on this?

**Comment by flying-sausages** [comment]
I'm down. IIRC the stats clarly showed that nginx is almost always installed together with panel.
